### DISCUSSION TITLE
Please review PRs!

### DISCUSSION THREAD

**Discussion by alejandro5042** [IviFoundation/ivifoundation.github.io#55] [General]
We really need a 2nd reviewer on all the PRs. @jiri-kominek's PR needs to wait on my PRs to go in, since his code builds on top of mine.

**Comment by joseph-mueller** [comment]
Alejandro, if it's cool with you, I can modify your PR as described in my comments and get it merged.   Probably some messiness with Jiri's because I think he touched a file I didn't think the auto conversion of H2's worked well in.

**Comment by jiri-kominek** [comment]
If my base is causing you a headache, I can re-branch it from anywhere else and do the merge.

**Comment by alejandro5042** [comment]
All PRs are reviewed! Good job everyone. Now just two PRs remain that are both drafts.
